doc/onboarding-extras.md

```diff
@@ -1,26 +1,24 @@
 ## Who to CC in issues
 
-* `lib/buffer`: @trevnorris
-* `lib/child_process`: @cjihrig, @bnoordhuis, @piscisaereus
-* `lib/cluster`: @cjihrig, @bnoordhuis, @piscisaereus
-* `lib/{crypto,tls,https}`: @indutny, @shigeki, @nodejs/crypto
-* `lib/domains`: @misterdjules
-* `lib/{_}http{*}`: @indutny, @bnoordhuis, @nodejs/http
-* `lib/net`: @indutny, @bnoordhuis, @piscisaereus, @chrisdickinson, @nodejs/streams
-* `lib/{_}stream{s|*}`: @nodejs/streams
-* `lib/repl`: @fishrock123
-* `lib/timers`: @fishrock123, @misterdjules
-* `lib/zlib`: @indutny, @bnoordhuis
-
-* `src/async-wrap.*`: @trevnorris
-* `src/node_crypto.*`: @indutny, @shigeki, @nodejs/crypto
-
-* `test/*`: @nodejs/testing, @trott
-
-* `tools/eslint`, `.eslintrc`: @silverwind, @trott
-
-* upgrading v8: @bnoordhuis / @targos / @ofrobots
-* upgrading npm: @thealphanerd, @fishrock123
+| subsystem | maintainers |
+| --- | --- |
+| `lib/buffer` | @trevnorris |
+| `lib/child_process` | @cjihrig, @bnoordhuis |
+| `lib/cluster` | @cjihrig, @bnoordhuis |
+| `lib/{crypto,tls,https}` | @nodejs/crypto |
+| `lib/domains` | @misterdjules |
+| `lib/{_}http{*}` | @indutny, @bnoordhuis, @mscdex, @nodejs/http |
+| `lib/net` | @indutny, @bnoordhuis, @nodejs/streams |
+| `lib/{_}stream{s|*}` | @nodejs/streams |
+| `lib/repl` | @fishrock123 |
+| `lib/timers` | @fishrock123, @misterdjules |
+| `lib/zlib` | @indutny, @bnoordhuis |
+| `src/async-wrap.*` | @trevnorris |
+| `src/node_crypto.*` | @nodejs/crypto |
+| `test/*` | @nodejs/testing, @trott |
+| `tools/eslint`, `.eslintrc` | @silverwind, @trott |
+| upgrading v8 | @bnoordhuis, @targos, @ofrobots |
+| upgrading npm | @thealphanerd, @fishrock123 |
```

Comment:
Would much prefer to make these all team references and begin moving away from specific individuals. Doing so has worked effectively for things like lts, documentation, streams, http, and crypto. For instance, I've created a @nodejs/buffer team as an example.

The downside to mentioning specific individuals is that it creates a disincentive for new collaborators to feel empowered to jump in and help with the reviews and puts too much emphasis and burden on specific individuals.